src/Native/System.Native/pal_runtimeinformation.h

```diff
@@ -14,5 +14,6 @@ enum
 {
     ARCH_X86,
     ARCH_X64,
-    ARCH_ARM
+    ARCH_ARM,
+    ARCH_ARM64
```

Comment:
> On managed side, the s_osArch will remain null and is going to throw InvalidOperationException for arm64.

I understand.  I'm saying it'll assert in debug and in release it'll throw what's effectively a null reference exception (though it's an InvalidOperationException because that's what was chosen for a nullable).  I'm suggesting that's not a particularly good experience... it looks like a coding flaw rather than intentional behavior.